metricbeat/helper/interfacer.go

```diff
@@ -32,6 +33,7 @@ type ModuleConfig struct {
 	MetricSets []string `config:"metricsets"`
 	Enabled    bool     `config:"enabled"`
 	Selectors  []string `config:"selectors"`
+	Filter     []filter.FilterConfig
```

Comment:
Yes. The idea is to allow filtering already on a more local level. So both can be used.